docs/src/pages/components/authenticator/customization.labels-and-text.web.mdx

```diff
@@ -1,14 +1,14 @@
 import { Authenticator, Button } from '@aws-amplify/ui-react';
 
 import { Example } from '@/components/Example';
-import { Tabs, TabItem } from '@aws-amplify/ui-react';
+import { Expander, ExpanderItem } from '@aws-amplify/ui-react';
 
 import { LabelsAndTextDemo } from './LabelsAndTextDemo';
 
 Using the same techniques as [Internationalization (I18n)](#internationalization-i18n), you can customize the labels and text of the components:
 
-<Tabs>
-  <TabItem title="Default Values">
+<Expander type="single" isCollapsible>
+  <ExpanderItem title="Default Values" value="default-values">
     Compare the default labels below to those in the customized screens via the tabs above.
```

Comment:
Since we don't have tabs anymore,

```suggestion
    Compare the default labels here to those in the customized below.
```

Reply:
Also, we should consider separating "Default Values" out of expanders because its purpose is to show reference to default values.